homeassistant/helpers/entity_registry.py

```diff
@@ -234,6 +235,11 @@ async def _async_migrate_func(
                     continue
                 entity["device_class"] = None
 
+        if old_major_version == 1 and old_minor_version < 9:
+            # Version 1.9 adds translation_key
+            for entity in data["entities"]:
+                entity["translation_key"] = entity.get("translation_key")
```

Comment:
You want to just set a value if it's not there. Do it like this:

```suggestion
                entity.setdefault("translation_key", None)
```

Reply:
Good point, I'll clean that up in a separate PR 👍